src/store/wallet.js

```diff
@@ -60,6 +62,7 @@ export default {
     exportFileLink: '',
     checkTokenGenerationEventDate: new Date(GENESIS_EVENT_TIMESTAMP),
     mainnetReady: false,
+    darkMode: false,
```

Comment:
With all the changes you made, we could add easily any theme. To make the most this, I would use a variable called "theme" witch value could be "light" | "dark". With this approach we could update or add a theme easily